scenario_player/utils/token.py

```diff
@@ -1,483 +1,182 @@
-
-            * token name
-            * deployment block
-            * contract address
-
-        And is stored as a JSONEncoded string::
-
-            '{"name": "<token name>", "address": "<contract address>", "block": <deployment block}'
-
-
-        """
-        token_data = {
-            "address": self.checksum_address,
-            "block": self.deployment_block,
-            "name": self.name,
-        }
+def eth_maybe_transfer(
+    orchestration_client: JSONRPCClient, target: Address, minimum_balance: int
+) -> None:
+    balance = orchestration_client.balance(target)
 
-        # Make sure to create the path, if it does not exist.
-        if not self._token_file.exists():
-            self._token_file.parent.mkdir(exist_ok=True, parents=True)
-            self._token_file.touch(exist_ok=True)
-
-        # Store the address and block number of this token contract on disk.
-        self._token_file.write_text(json.dumps(token_data))
-
-    def init(self):
-        """Load an existing or deploy a new token contract."""
-        if self.config.token.reuse_token or self.config.token.address:
-            return self.use_existing()
-        return self.deploy_new()
-
-    def use_existing(self) -> Tuple[str, int]:
-        """Reuse an existing token, loading its data from the scenario's `token.info` file.
-
-        :raises TokenSourceCodeDoesNotExist:
-            If no source code is present at the loaded address.
-        """
-        if self.config.token.reuse_token:
-            token_data = self.load_from_file()
-            contract_name, address, block = (
-                token_data["name"],
-                token_data["address"],
-                token_data["block"],
-            )
-        elif self.config.token.address:
-            contract_name = CUSTOM_TOKEN_NAME
-            address = self.config.token.address
-            block = 1
-        else:
-            raise RuntimeError("`use_existing()` called without `reuse` or `address` set.")
-
-        try:
-            check_address_has_code(
-                client=self._local_rpc_client,
-                address=address,
-                contract_name=contract_name,
-                given_block_identifier="latest",
-            )
-        except AddressWithoutCode as e:
-            raise TokenSourceCodeDoesNotExist(
-                f"Cannot reuse token - address {address} has no code stored!"
-            ) from e
-
-        # Fetch the token's contract_info data.
-        contract_info = self._local_contract_manager.get_contract(contract_name)
-
-        self.contract_data = {
-            "token_contract": address,
-            "name": contract_info.get("name") or contract_name,
-        }
-        self.contract_proxy = self._local_rpc_client.new_contract_proxy(
-            contract_info["abi"], address
+    if balance < minimum_balance:
+        eth_transfer = EthTransfer(
+            to_address=target,
+            value=minimum_balance - balance,
```

Comment:
This used to fund to `NODE_ACCOUNT_BALANCE_FUND` so we don't have to re-fund on every run in the reuse case.

Reply:
That didn't change:

https://github.com/raiden-network/scenario-player/blob/e363a5bc9663d23d65ce0adc84f24525d9868771/scenario_player/runner.py#L500

Reply:
Oh, wait, these are different constants.